client/js/stores/snippet-store.js

```diff
@@ -23,6 +23,28 @@ class SnippetStore {
     }
   }
 
+  sortSnippets(data) {
+    if(data.ok){
+      let sortMethod = data.sortMethod;
+      const {snippets} = this.state;
+      let newSnippets = _.extend([], snippets);
+      if(sortMethod == 'date'){
+        newSnippets.sort( function(a,b) {
+          var aDate = new Date(a.createdAt).getTime();
+          var bDate = new Date(b.createdAt).getTime();
+          return (bDate > aDate) ? 1 : ((aDate > b) ? -1 : 0);
+        }); 
+      } else {
```

Comment:
we do. there's `if` for 'date' and else for `avg`. i didn't specify it in ```else if(sortMethod == 'avg)```, but should I?